pkgs/development/python-modules/qasync/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyside2
+, pytest
+}:
+
+buildPythonPackage rec {
+  version =  "0.23.0";
+  src = fetchFromGitHub {
+    owner = "CabbageDevelopment";
+    repo = "qasync";
+    rev = "58882735229b0d17836621d7d09ce02a6f80789d";
+    sha256 = "sha256-2wustBtShydCXM5L5IQbOaZ2BfGxbIPwLZ8sRfxFnM4=";
+  };
+
+  pname = "qasync";
+
+  propagatedBuildInputs = [ pyside2 ];
+
+  # sigabrt
+  # checkInputs = [pytest];
+  # checkPhase = ''
+  #   pytest
+  # '';
+
+  doCheck = false;
```

Comment:
Please try using [pytestCheckHook](https://nixos.org/manual/nixpkgs/stable/#using-pytestcheckhook), or comment why the tests are skipped. If you still chose skip it, please remove the commented checkPhase code.